docs/resources/aws_batch_job_definition.md

```diff
@@ -0,0 +1,109 @@
+| Property | Description|
+| --- | --- |
+| job_definition_name | The name of the job definition. |
+| job_definition_arn | The arn of the job definition. |
+| revision | The revision of the job definition. |
+| status | The status of the job definition. |
+| type | The type of the job definition. |
+| parameters | The parameters of the job definition. |
+| container_properties (image) | The image of the container properties of the job definition. |
+| container_properties (vcpus) | The vcpus of the container properties of the job definition. |
+| container_properties (memory) | The memory of the container properties of the job definition. |
+| container_properties (command) | The command of the container properties of the job definition. |
+| container_properties (job_role_arn) | The job role arn of the container properties of the job definition. |
+| container_properties (execution_role_arn) | The execution role arn of the container properties of the job definition. |
+| container_properties (volumes) | The volumes of the container properties of the job definition. |
+| container_properties (environment) | The environment of the container properties of the job definition. |
+| container_properties (mount_points) | The mount points of the container properties of the job definition. |
+| container_properties (readonly_root_filesystem) | The image of the container properties of the job definition. |
+| container_properties (image) | The readonly root filesystem of the container properties of the job definition. |
+| container_properties (privileged) | The privileged of the container properties of the job definition. |
+| container_properties (ulimits) | The ulimits of the container properties of the job definition. |
+| container_properties (user) | The user of the container properties of the job definition. |
+| container_properties (instance_type) | The instance type of the container properties of the job definition. |
+| container_properties (resource_requirements) | The resource requirements of the container properties of the job definition. |
+| container_properties (linux_parameters (shared_memory_size)) | The shared_memory_size of the linux parameters of the container properties of the job definition. |
+| container_properties (linux_parameters (tmpfs)) | The tmpfs of the linux parameters of the container properties of the job definition. |
+| container_properties (linux_parameters (max_swap)) | The max_swap of the linux parameters of the container properties of the job definition. |
+| container_properties (linux_parameters (swappiness)) | The swappiness of the linux parameters of the container properties of the job definition. |
+| container_properties (log_configuration (options)) | The options of the log configuration of the container properties of the job definition. |
+| container_properties (log_configuration (shared_memory_size)) | The shared_memory_size of the log configuration of the container properties of the job definition. |
```

Comment:
Is shared_memory_size a log configuration or Linux parameter? It looks like Linux.

Reply:
Yes, shared_memory_size is a Linux parameter.